pkgs/by-name/li/libpwquality/package.nix

```
@@ -30,7 +32,7 @@ stdenv.mkDerivation rec {
   src = fetchFromGitHub {
     owner = "libpwquality";
     repo = "libpwquality";
-    rev = "${pname}-${version}";
+    tag = "${finalAttrs.pname}-${finalAttrs.version}";
```

```suggestion
    tag = "libpwquality-${finalAttrs.version}";
```
This is an anti pattern